extensions/cdpath

```diff
@@ -1,2 +1,3 @@
 # Note that tab completion doesn't work with CDPATH
-export CDPATH=.:~:~/work
```

Comment:
@jpf by the way, this breaks scripts.. because,   `SOMETHING=$(cd /tmp ; echo foo)` will set SOMETHING to `/tmp foo` due to CDPATH's behavior (POSIX compatibility!)

Reply:
If it comes to a vote, I'm outnumbered :D https://github.com/jpf/.profile.d/graphs/contributors

Reply:
![image](https://user-images.githubusercontent.com/855219/97476688-b74d3080-191c-11eb-91b1-6488a62d87b9.png)
Wait a second…  one of them is partly obscured, but looks like it might be sock puppet!! :)